tst/EnergyPlus/unit/UnitaryHybridAirConditioner.unit.cc

```diff
@@ -1,1565 +1,1565 @@
-// EnergyPlus, Copyright (c) 1996-2021, The Board of Trustees of the University of Illinois,
```

Comment:
I changed 3 lines of code in this unit test file. Good luck finding them. Something is going on with Clang.

Reply:
Nope, I just changed the VRF unit test file. @Myoldmopar might want to check if line endings is still and issue in UnitaryHybridAirConditioner.unit.cc